.github/workflows/deploy.yml

```diff
@@ -0,0 +1,112 @@
+      - name: Create OIDC Token
+        id: create-oidc-token
+        shell: bash
+        run: |
+          export OIDC_URL_WITH_AUDIENCE="$ACTIONS_ID_TOKEN_REQUEST_URL&audience=${{ secrets.K8S_AUDIENCE }}"
+          IDTOKEN=$(curl -H "Authorization: Bearer $ACTIONS_ID_TOKEN_REQUEST_TOKEN" \
+          -H "Accept: application/json; api-version=2.0" "$OIDC_URL_WITH_AUDIENCE" \
+          | jq -r .value)
+          echo "::add-mask::${IDTOKEN}"
+          echo "idToken=${IDTOKEN}" >>$GITHUB_OUTPUT
+      - name: Setup Kube Context
+        uses: azure/k8s-set-context@v2
+        with:
+          method: kubeconfig
+          kubeconfig: |
+            kind: Config
+            apiVersion: v1
+            current-context: default
+            clusters:
+            - name: default
+              cluster:
+                certificate-authority-data: ${{ secrets.K8S_CERTIFICATE_AUTHORITY_DATA }}
+                server: ${{ secrets.K8S_SERVER }}
+            users:
+            - name: oidc-token
+              user:
+                token: ${{ steps.create-oidc-token.outputs.IDTOKEN }}
+            contexts:
+            - name: default
+              context:
+                cluster: default
+                namespace: ${{secrets.K8S_NAMESPACE}}
+                user: oidc-token
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.IAM_ROLE }}
+          role-session-name: GitHub_to_AWS_via_FederatedOIDC
+          aws-region: us-east-1
+      - name: Setup psql client
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y postgresql-client 
+      - name: Reinitialize Shoppy DWH DB
+        env:
+          DB_URL: postgres://${{ secrets.SHOPPY_DB_USER }}:${{ secrets.SHOPPY_DB_PASSWORD }}@${{ secrets.SHOPPY_DB_HOST }}:${{ vars.SHOPPY_DB_PORT }}/${{ vars.SHOPPY_DB }}
+        run: |
+          cd api &&
+          yarn db:initialize
+      - name: Download PostgreSQL dump from S3
+        run: |
+          aws s3 cp s3://${{ secrets.S3_BUCKET }}/${{ vars.S3_KEY }} dump.sql       
+      - name: Pause Metabase (set pause.enabled=true)
+        run: |
+          kubectl patch metabase hosting-shoppy \
+          --type=merge \
+          -p '{"spec":{"pause":{"enabled":true}}}'
+      - name: Drop app database and restore
+        run: |
+          export PGPASSWORD='${{ secrets.DB_PASSWORD }}'
+          export PGUSER=${{ secrets.DB_USER }}
+          export PGDATABASE=${{ secrets.DB_DATABASE }}
+          export PGHOST=${{ secrets.DB_HOST }}
+          psql -d ${{ secrets.DB_DATABASE }} \
+          -c "DROP DATABASE IF EXISTS ${{ secrets.DB_SHOPPY_DATABASE }} WITH (FORCE);"
+          psql -d ${{ secrets.DB_DATABASE }} \
+          -c "CREATE DATABASE ${{ secrets.DB_SHOPPY_DATABASE }} WITH OWNER ${{ secrets.DB_USER }};"
+          pg_restore -d ${{ secrets.DB_SHOPPY_DATABASE }} dump.sql
```

Comment:
```
pg_restore -d ${{ secrets.DB_SHOPPY_DATABASE }} dump.sql
```
when i run the same command in docker, i have:
`pg_restore: error: input file appears to be a text format dump. Please use psql.`

Are we sure that `pg_restore -d ${{ secrets.DB_SHOPPY_DATABASE }} dump.sql` will work?

Reply:
Nvm, on CI it works

Reply:
It requires the correct flags when the dump is created.